index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="pages/styles.css" rel="stylesheet"/>
+  <title>Megan Hagar's Portfoilio</title>
+</head>
+<body>
+  <section id="nav-background">
+    <nav>
+      <ul id="navbar">
+        <li> </li>
+        <li> </li>
+        <li> </li>
+        <li> </li>
+      </ul>
+    </nav>
+  </section>
+  <main id="index">
+    <header>
+      <h1>Welcome to my profile!</h1>
+      <h2>My name is Megan Hagar, I'm currently a student at Ada Developer's Academy.</h2>
+    </header>
+    <p>Previously working in the marketing and sales world for a local cruise line, I decided to pivot my career to software engineering.</p>
+    <section id="bubbles">
+      <section id="bubbles-grid">
+        <section class="bubble">
+          <a href="/pages/portfolio.html">Portfolio</a>
+        </section>
+        <section class="bubble">
+          <a href="/pages/about.html">About</a>
+        </section>
+        <section class="bubble">
+          <a href="/pages/blog.html">Blog</a>
+        </section>
+      </section>
+    </section>
+  </main>
+  <footer>
+    <h4>2021 Megan Hagar</h4>
```

Comment:
Consider using a tag other than `h4` here. It's not really a subheading nested 4 deep, so this isn't quite semantic. Instead, you can use CSS to change the font size.